.travis.yml

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+dist: trusty
+
+language: c
+
+before_install:
+  - sudo apt-get update
+
+install:
+  - sudo apt-get install gcc-arm-none-eabi
```

Comment:
I'd like to recommend that we use the launchpad toolchain from here: https://launchpad.net/gcc-arm-embedded in which case we can use:
```
  - sudo add-apt-repository -y ppa:terry.guo/gcc-arm-embedded
  - sudo apt-get install -y --force-yes gcc-arm-none-eabi
```
I'm not sure if the --force is required. I'd also recommend that we add a line:
```
- arm-none-eabi-gcc --version
```
so that we know which version of the toolchain is actually being used by Travis.